### PR TITLE
unit test and documentation for deploy uncustomised server

### PR DESCRIPTION
**Pull request by samuelchong**
### Description
Documentation and unit test for deploy uncustomised server

### Status

Replace this: describe the PR status. Examples:
- done, ready for review

### Checklist (tick everything that applies)

- [X] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [X] Documentation
- [X] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

File: libcloud/compute/drivers/dimensiondata.py [hunk 1]
```diff
@@ -3830,6 +3830,66 @@ def ex_create_node_uncustomized(self,
 
         Create Node in MCP2 Data Center
 
+        >>> from pprint import pprint
```
Comment (tonybaloney) [inline libcloud/compute/drivers/dimensiondata.py]: this would be better in docs/examples

**Review by tonybaloney — CHANGES_REQUESTED** [review]

File: libcloud/compute/drivers/dimensiondata.py (hunk 1 above)
Comment (samuelchong) [reply libcloud/compute/drivers/dimensiondata.py]: Agree. I've moved created a folder for dd in docs/examples. Sample code moved to folder.

File: docs/examples/compute/dimensiondata/Nodes_Create_mcp2_Uncustomised.py [hunk 2]
```diff
@@ -6,15 +6,16 @@
 # Get dimension data driver
 libcloud.security.VERIFY_SSL_CERT = True
```
Comment (tonybaloney) [inline docs/examples/compute/dimensiondata/Nodes_Create_mcp2_Uncustomised.py:7]: This is default anyway. But not that important.

**Comment by tonybaloney** [comment]
👍

**Pull request closed without merging**